framework/src/arastorage/storage_interface.c

```diff
@@ -84,9 +84,14 @@ struct index_record_s {
 };
 
 /****************************************************************************
-* Public Functions
+* Public Variables
 ****************************************************************************/
 #ifdef CONFIG_ARASTORAGE_ENABLE_WRITE_BUFFER
+static struct insert_buffer_s g_storage_write_buffer;
```

Comment:
It's `static` global variable. Then, is it `public` or `private` variable?

Reply:
I think this will come under private data

Reply:
Forgot to change the heading after adding static. Thanks for pointing it out Mr. Chang. Modified.